tst/EnergyPlus/unit/LowTempRadiantSystem.unit.cc

```diff
@@ -0,0 +1,256 @@
+// EnergyPlus::Low Temperature Radiant Unit Tests
+
+// Google Test Headers
+#include <gtest/gtest.h>
+
+// EnergyPlus Headers
+#include <EnergyPlus/LowTempRadiantSystem.hh>
+#include <EnergyPlus/DataHeatBalance.hh>
+#include <EnergyPlus/DataPlant.hh>
+#include <EnergyPlus/DataZoneEquipment.hh>
+#include <EnergyPlus/DataSizing.hh>
+#include <EnergyPlus/FluidProperties.hh>
+#include <EnergyPlus/UtilityRoutines.hh>
+#include <ObjexxFCL/gio.hh>
+
+
+using namespace EnergyPlus;
+using namespace EnergyPlus::LowTempRadiantSystem;
+using namespace ObjexxFCL;
+using namespace EnergyPlus::DataHeatBalance;
+using namespace DataGlobals;
+using namespace EnergyPlus::DataPlant;
+using namespace EnergyPlus::DataZoneEquipment;
+using namespace EnergyPlus::DataSizing;
+using namespace EnergyPlus::FluidProperties;
+
+
+class LowTempRadiantSystemTest : public ::testing::Test
+{
+
+public:
+
+	int RadSysNum;
```

Comment:
@Myoldmopar What am I doing wrong here?  I tried to model other unit tests with the class setup/teardown and TEST_F structure.  But when I use RadSysNum as an argument to SizeLowTempRadiantSystem it does not pass the value properly.  In debug build, the argument in SizeLowTempRadiantSystem bounces around upon initial entry to the routine but then reverts to 1 (which is should be) by the time it's used later in the routine.  In release mode, the argument is some random negative number while in SizeLowTempRadiantSystem.